packages/@aws-cdk/aws-greengrass/package.json

```diff
@@ -30,7 +30,14 @@
         ]
       }
     },
-    "projectReferences": true
+    "projectReferences": true,
+    "metadata": {
+      "jsii": {
+        "rosetta": {
+          "strict": true
+        }
```

Comment:
can we get rid of the addtional 'metadata' and 'jsii' subsections here?

Keep it simple as so -

```
{
  name: '...'
  jsii: {
    python: ...,
    rosetta: {
      strict: true
    }
  }
}
```

Reply:
This is how strict mode is to be configured, per the `jsii-rosetta` contract. The fact it's buried in metadata here is actually the "standard" way `jsii` provides to provide assembly-level tool-specific configuration.

Reply:
Seems very icky to me, esp the unnatural deep nesting and the duplicated `jsii` key.

Do you agree that the suggestion I've provided (or some variant of it) is better than this? 
If so and if fixing now is going to make it worse, can we open an issue to track improving the "standard" way?

Reply:
The current situation is a "hack". The long term strategy is for this behaviour to be the default behaviour, however this is a breaking change. This is meant to be icky so we get rid of it on new major version...

There had been discussions over the PR that introduced this configuration point... It was decided that this is what make sense. Changing how such metadata is registered (including the existing use-case sin the wild) is also a breaking change.